main/res/values/strings.xml

```diff
@@ -510,8 +510,8 @@
     <string name="confirm_data_dir_move">Moving the geocache data might take a really long time, depending on your device and the amount of data.</string>
 
     <string name="settings_title_basicmembers">Options for Basic Members</string>
-    <string name="settings_title_navigation">Navigation</string>
-    <string name="settings_summary_navigation">Choose preferred navigation methods and select external navigation apps.</string>
+    <string name="settings_title_navigation">Offline routing / Navigation</string>
```

Comment:
I would rather suggest "Navigation / Routing" to 
1. keep the known one in front
2. keep it a bit shorter (offline is mentioned in the subtitle anyway)

Reply:
Compromise: "Routing / Navigation" ?